safety_limiter/test/src/test_safety_limiter.cpp

```diff
@@ -609,6 +609,88 @@ TEST_F(SafetyLimiterTest, SafetyLimitMaxVelocitiesValues)
   }
 }
 
+TEST_F(SafetyLimiterTest, SafetyLimitOmniDirectional)
+{
+  ros::Rate wait(20.0);
+  const double vel = 1.5;
+  const double threshold_angle = std::atan2(1.0, 0.1);
+  for (double angle = -M_PI; angle < M_PI; angle += M_PI / 8)
+  {
+    for (int i = 0; i < 10 && ros::ok(); ++i)
+    {
+      publishSinglePointPointcloud2(2.0, 0, 0, "base_link", ros::Time::now());
+      publishWatchdogReset();
+      publishTwist(0, 0);
+      wait.sleep();
+      ros::spinOnce();
+    }
+
+    double obstacle_x;
+    double obstacle_y;
+    if (std::abs(angle) < threshold_angle)
+    {
+      // Moving forward
+      obstacle_x = 0.5 * std::cos(angle);
+      obstacle_y = 0.5 * std::sin(angle);
+    }
+    else if ((M_PI - threshold_angle) < std::abs(angle))
+    {
+      // Moving backward
+      obstacle_x = 0.5 * std::cos(angle) - 2.0;
+      obstacle_y = 0.5 * std::sin(angle);
+    }
+    else if (angle > 0)
+    {
+      // Moving left
+      obstacle_x = 0.5 * std::cos(angle) - 1.0;
+      obstacle_y = 0.5 * std::sin(angle) + 0.1;
+    }
+    else
+    {
+      // Moving right
+      obstacle_x = 0.5 * std::cos(angle) - 1.0;
+      obstacle_y = 0.5 * std::sin(angle) - 0.1;
+    }
+
+    // 1.5 m/ss, obstacle at 0.5 m: limited to 1.0 m/s
```

Comment:
Sorry, 1.0 m/ss acceleration was correct
https://github.com/at-wat/neonavigation/blob/20a900107ff2c9f180c6e37eba0d6831846971d9/safety_limiter/test/test/safety_limiter_rostest.test#L17
```suggestion
    // 1.0 m/ss, obstacle at 0.5 m: limited to 1.0 m/s
```